string text  #text to say. See sound_player.py for list of supported commands.
